src/app/shared/spaces.service.mock.ts

```diff
@@ -0,0 +1,155 @@
+import { cloneDeep } from 'lodash';
+import {
+  Space,
+  Context,
+  Contexts,
+  ContextTypes,
+  SpaceService,
+  SpaceNamePipe
+} from 'ngx-fabric8-wit';
+
+
+export const context1: Context = {
+  user: {
+    "attributes": {
+      "fullName": "Corinne Krych",
+      "imageURL": "https://www.gravatar.com/avatar/d4445894370ad3aeb93345b544ba1dbf.jpg",
+      "username": "ckrych@redhat.com"
```

Comment:
Does @corinnekrych know that you are adding her into the code like this?

Reply:
No. I referred mock data from other test cases.
Also I thought username and other credentials are just place holders for testing purpose.